### ISSUE TITLE
Completion popup shouldn't be moved when scrolling

### ISSUE THREAD

**Issue by Gama11** [fdorg/flashdevelop#760]
![scroll](https://cloud.githubusercontent.com/assets/2620907/8039367/7ba3a38a-0e09-11e5-9f25-7cba9eb4d7b1.gif)

It should stay at that same line / not be affected by scrolling.

**Comment by Neverbirth** [comment]
I have this already covered. I'll commit the PR once this PR is merged https://github.com/fdorg/flashdevelop/pull/732

**Comment by Gama11** [comment]
There's a similar issue with zooming in and out.

**Comment by Neverbirth** [comment]
Yeah, those are covered as well IIRC.
